includes/classes/db/mysql/define_queries.php

```diff
@@ -31,5 +31,5 @@
                      AND p.products_status = 1
                      GROUP BY p.products_id, p.products_image
                      ORDER BY date_purchased desc, p.products_id
-                     LIMIT 50");
+                     LIMIT %s");

```

Comment:
For safety, this should probably be `%u` ('integer, treated as unsigned number')